common/values/kustomization.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+configMapGenerator:
+  - name: kubernetes-dashboard-6.0.0-values
```

Each application has a d2iq defaults configmap called something like `kubernetes-dashboard-6.0.0-d2iq-defaults`. 

So maybe a more descriptive name for these would be `kubernetes-dashboard-6.0.0-combined-defaults`, WDYT?
```suggestion
  - name: kubernetes-dashboard-6.0.0-combined-defaults
```